commands/commands.go

```diff
@@ -55,8 +60,13 @@ func (c *Command) Run(pctx context.Context, bus *events.EventBus) {
 	log.Debugf("%s.Run start", c.Name)
 
 	cmd := ArgsToCmd(c.Exec, c.Args)
```

Comment:
Nit, I realize `ArgsToCmd` isn't a part of this commit but I had to look up the implementation in order to discover that its better represented as `ExecWithArgs`. Even if the exec may not happen with args. `ArgsToCmd` sounds like message passing or type conversion.